src/client/dfuse/ops/lookup.c

```diff
@@ -256,7 +256,7 @@ check_for_uns_ep(struct dfuse_projection_info *fs_handle,
 	if (rc)
 		D_GOTO(out_umount, ret = rc);
 
-	rc = dfuse_lookup_inode(fs_handle, dfs, &oid,
+	rc = dfuse_lookup_inode(fs_handle, dfs, NULL,
```

Comment:
What does this do?

Reply:
This is a bug-fix to dfuse, if a path is accessed via the container path, and also through an unified namespace entry-point it's entered into the hash table with two different sets of options, so is marked as two different inodes, which can then lead to access-after-free or similar during forget calls from the kernel.